docs/configuration/index.md

```diff
@@ -636,6 +636,18 @@ Prior to version 0.13.0, Druid string columns treated `''` and `null` values as
 
 This mode does have a storage size and query performance cost, see [segment documentation](../design/segments.md#sql-compatible-null-handling) for more details.
 
+### HTTP Client
+
+All Druid components can communicate with each other over HTTP.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.global.http.numConnections`|Size of connection pool per destination URL. If there are more HTTP requests than this number that all need to speak to the same URL, then they will queue up.|20|
+|`druid.global.http.compressionCodec`|Compression codec to communicate with others. May be "gzip" or "identity".|gzip|
+|`druid.global.http.readTimeout`|The timeout for data reads.|PT15M|
```

Comment:
nit: for default values column it looks like you have a mix of using `backticks` and not

Reply:
Thanks. Wrapped default values.